internal/pkg/cmd/prerunner.go

```diff
@@ -863,47 +861,33 @@ func (r *PreRun) HasAPIKey(command *HasAPIKeyCLICommand) func(*cobra.Command, []
 	}
 }
 
-func (r *PreRun) ValidateToken(config *dynamicconfig.DynamicConfig, unsafeTrace bool) error {
+func (r *PreRun) ValidateToken(config *dynamicconfig.DynamicConfig) error {
 	if config == nil {
 		return new(errors.NotLoggedInError)
 	}
 	ctx := config.Context()
 	if ctx == nil {
 		return new(errors.NotLoggedInError)
 	}
-	err := r.JWTValidator.Validate(ctx.Context)
-	if err == nil {
-		return nil
-	}
-	switch err.(type) {
-	case *ccloudv1.InvalidTokenError:
-		return r.updateToken(new(ccloudv1.InvalidTokenError), ctx, unsafeTrace)
-	case *ccloudv1.ExpiredTokenError:
-		return r.updateToken(new(ccloudv1.ExpiredTokenError), ctx, unsafeTrace)
-	}
-	if err.Error() == errors.MalformedJWTNoExprErrorMsg {
-		return r.updateToken(errors.New(errors.MalformedJWTNoExprErrorMsg), ctx, unsafeTrace)
-	} else {
-		return r.updateToken(err, ctx, unsafeTrace)
-	}
+	return r.JWTValidator.Validate(ctx.Context)
```

Comment:
how does this handle invalid token / expired token?

Reply:
You can take a look at the implementation! It checks for the existence of the `exp` field and if it's expired.